src/app/dim-ui/CollapsibleTitle.scss

```diff
@@ -1,6 +1,25 @@
 @use '../variables.scss' as *;
 
 .title {
+  text-align: center;
+  appearance: none;
+  background: transparent;
+  border: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+  font-family: inherit;
+
+  > button {
+    all: initial;
+    font-family: inherit;
+    cursor: pointer;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    flex: 1;
+  }
+
```

Comment:
Worth including a focus style 

```suggestion
.title:has(button:focus) {
    box-shadow: inset 0 0 0 1px var(--theme-accent-primary);
}
```

Reply:
Just noticed hover styles are failing to kick in — these should probably be consolidated with focus styles so that mouse + keyboard interactions have the same visual cue. 

Atm there are 'dead zones' when clicking on the space between `button` elements (even though the cursor is a pointer). To avoid this, either the `button` should be 'full bleed' inside the `h3`  or the hover/focus styles should only apply to the interactive region. 

![image](https://github.com/DestinyItemManager/DIM/assets/1396158/77dc6398-9e36-4ff1-9524-238fde6c2ce0)

Reply:
Now I'm remembering why I'd stopped working on this :-)